docker/tests/binaries.json

```diff
@@ -0,0 +1,40 @@
+{
+    "released": [
```

Comment:
Btw I wonder if we can keep this file up-to-date automatically since there is a chance that we forget to update it

I don't insist on doing it in this PR, we always can open a follow-up one

Reply:
I think we can. Sounds like a simple script, and a small github action that updates list on release

Reply:
Sounds good

Could you please open a follow-up issue for this after merging this PR?